<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEvenement</class>
 <widget class="QDialog" name="DialogEvenement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="idEVE_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_29">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>80</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>pdf</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_36">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>170</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>recherche </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_30">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>110</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>print</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NBpEVE_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>140</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>id</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>type</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>date</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>nbr</string>
    </property>
   </item>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget_8">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>340</y>
     <width>131</width>
     <height>141</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_33">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>430</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>modifier</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>50</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>supprimer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rechercheM_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>140</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_21">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>400</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>affichage</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idES_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="TypeEVE_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>stat</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
